wdl/PETestChromosome.wdl

```diff
@@ -215,8 +221,9 @@ task PETest {
       done
       echo "PE-tabix retry:" $x
       cmp --silent PE_1.txt.gz PE_2.txt.gz || exit 1
-    
+
       mv PE_1.txt.gz PE.txt.gz
+      rm PE_2.txt.gz
```

Comment:
Can you remove these changes? We are replacing this code and this will cause a conflict.

Reply:
Done